Makefile

```diff
@@ -130,6 +130,7 @@ $(ISO): $(ALL)
 		-input-charset utf-8 \
 		-output-charset utf-8 \
 		$(ISO_ROOT)
+	sha256sum $@ | cut -d" " -f 1 > $@.sha256sum
```

Comment:
This way, you're not producing a valid checksum file (i.e. one that can be used as input of `sha256sum --check`). Instead:

```
cd $(dirname $@) && sha256sum $(notdir $@) > SHA256SUM
```
and make sure that file (`SHA256SUM`) gets uploaded.